tests/utils/insert-inner-block.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import { canvas } from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import { selectBlockByName } from '.';
+
+/**
+ * Inserts an inner block into the currently selected block. If a parent block
+ * is provided, it will be selected before inserting the inner block.
+ *
+ * We make the parentBlockName optional to add more flexibility to the function,
+ * enabling tests to have more control over the selection of the parent block.
+ *
+ * @param  blockTitle      Block title, such as "Add to Cart Button".
+ * @param  parentBlockName Parent block name, such as core/group.
+ */
+export const insertInnerBlock = async (
+	blockTitle: string,
+	parentBlockName = ''
+) => {
+	if ( parentBlockName ) {
+		await selectBlockByName( parentBlockName );
+	}
+	const blockInserterButton = await canvas().waitForSelector(
+		'.block-editor-inserter button'
```

Comment:
I'm always for refactoring these selectors somewhere else. These might change in Gutenberg and we need one easy place to make sure we can edit all the functions which uses them. It also makes them more explicit when reading the code.

We have a `SELECTORS` object inside `tests/e2e/utils.js`. If you want to create a separate file for this function, maybe you can `export` those selectors from that file? Or, actually better: if we want to make one file per function, then we should have one file for the selectors.

Reply:
> Or, actually better: if we want to make one file per function, then we should have one file for the selectors.

I prefer this too, implemented in [ac1222c](https://github.com/woocommerce/woocommerce-blocks/pull/7482/commits/ac1222c7c6e38bb8d61fab21c2846683207e05ea)